albatross/core/model.h

```diff
@@ -256,6 +259,7 @@ class RegressionModel : public ParameterHandlingMixin {
   }
 
   bool has_been_fit_;
+  std::map<std::string, std::string> metadata_;
```

Comment:
nit: Should we call this `insights` to make it consistent and clear these are results from the model fit, not metadata of the inputs?

Reply:
yeah good call, I like that name better.